recipes/pam.rb

```diff
@@ -71,7 +72,7 @@
   end
 
   # configure tally2
-  if node['os-hardening']['auth']['retries'] > 0
+  if node['os-hardening']['auth']['retries'].positive?
```

Comment:
do you know, why there is the error with the ubuntu 16.04 machine with chef 12.5